types/testingutils/ssv_msgs.go

```diff
@@ -52,7 +54,7 @@ var TestAggregatorConsensusDataByts, _ = TestAggregatorConsensusData.Encode()
 
 var TestProposerConsensusData = &types.ConsensusData{
 	Duty:    TestingProposerDuty,
-	Version: spec2.DataVersionBellatrix,
+	Version: spec.DataVersionBellatrix,
```

Comment:
This whole file will be deleted and this is just temporary?

Reply:
this file should not be deleted, there is a logic besides proposals